asciidoc/modules/con_mg_changes-in-json.adoc

```diff
@@ -118,7 +137,39 @@ Use the following example code to convert a Base64URL to Base64 encoder.
 [source,java,options="nowrap",subs="attributes+"]
 ----
 String base64url = someJsonObject.getString("base64encodedElement")
-String base64 = Json.toBase64(base64url);
+String base64 = toBase64(base64url);
+----
+
+The helper functions described above address plain JSON migrations. If you use automatic object mapping from JSON
+to a {Java} POJO, then a custom object mapper needs to be created to your specific use case.
+
```

Comment:
@pmlopes Can we document the helper function names? Also again is my understanding correct?

The helper functions XXX in the utility enable only JSON migrations. If you use automatic object mapping from JSON
to a {Java} POJO in your applications, then you must create a custom object mapper to convert the Base64
string to Base64URL.